recipesearch/src/Recipes/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import './SearchBar.css';
+import  {searchRecipesAction} from './action';
+import { connect } from 'react-redux';
+
+export class SearchBar extends Component {
+    state = {
+        searchValue: '',
+    }
+
+    handleSearchChange = (event) => {
+        this.setState({
+            searchValue: event.target.value,
+        })
+    }
+
+    handleRecipeSearch = (event) =>{
+        const { searchValue } = this.state
+        event.preventDefault();
+        this.props.searchRecipesAction(searchValue);
```

Comment:
You should get into the habit of always destructuring props though in this case, `this.props...` isn't too bad

Reply:
Alright @edmondatto I sure will